### PR TITLE
Goal overlay

### PR DESCRIPTION
**Pull request by pflopez**
Screen Shot 2024-03-19 at 14 43 30

### CODE REVIEW

File: src/interface/src/app/plan/create-scenarios/create-scenarios.component.ts [hunk 1]
```diff
@@ -240,37 +243,6 @@ export class CreateScenariosComponent implements OnInit {
         this.selectedTab = ScenarioTabs.RESULTS;
         this.pollForChanges();
       });
-
-    // this.createUploadedProjectAreas()
```
Comment (pflopez) [inline src/interface/src/app/plan/create-scenarios/create-scenarios.component.ts:244]: This is unrelated to the pr, just commented code that doesn't apply

File: src/interface/src/app/plan/create-scenarios/goal-overlay/goal-overlay.component.html [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  </header>
+  <section>
+    <p>{{ goal?.long_question_text }}</p>
+    <p *ngFor="let p of goal?.description" [innerHTML]="p"></p>
+    <p>
+      For more information, see the
+      <a
+        href="https://www.planscape.org/documentation/user-guide/"
+        target="_blank"
+        >User Guide</a
```
Comment (lastminutediorama) [inline src/interface/src/app/plan/create-scenarios/goal-overlay/goal-overlay.component.html:16]: Will all of the links point to the same place for each goal, or are there going to be individual links for them at some point?

Comment (pflopez) [reply src/interface/src/app/plan/create-scenarios/goal-overlay/goal-overlay.component.html:16]: the one on the doc shared is the same for all

Comment (pflopez) [reply src/interface/src/app/plan/create-scenarios/goal-overlay/goal-overlay.component.html:16]: also the guide does not have anchors/links to individual sections :( (might be worth adding!)

File: src/interface/src/app/plan/create-scenarios/goal-overlay/goal-overlay.component.scss [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  @include public-sans();
+  display: block;
+  max-width: 540px;
+  position: absolute;
+  top: 110px;
+  left: 0;
+  z-index: 200;
+  margin: 0 8px;
+
+  @media screen and (max-width: 800px) {
```
Comment (lastminutediorama) [inline src/interface/src/app/plan/create-scenarios/goal-overlay/goal-overlay.component.scss:14]: I've been so tempted to add media queries, so I fully support this.
(At some point -- not in this PR -- maybe we could chat about mixins for breakpoints...)

**Review by lastminutediorama — APPROVED** [review]
👍

File: src/interface/src/app/plan/create-scenarios/goal-overlay/goal-overlay.component.scss (hunk 3 above)
Comment (pflopez) [reply src/interface/src/app/plan/create-scenarios/goal-overlay/goal-overlay.component.scss:14]: ha sure thing, these ones are  mostly so the modal doesn't completely break the ui when it runs out of place....
